plugins/modules/sonic_login_lockout.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2023 Dell Inc. or its subsidiaries. All Rights Reserved
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+
+"""
+The module file for sonic_login_lockout
+"""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community',
+    'license': 'Apache 2.0'
+}
+
+DOCUMENTATION = """
+---
+module: sonic_login_lockout
+version_added: '4.2.0'
```

Comment:
Please change the collection version
```suggestion
version_added: '2.4.0'
```

Reply:
Please use 2.5.0 (here and other applicable places) as the version number for the enterprise_sonic Ansible resource module collection.

Reply:
Accepted. Done with the changes